Cargo.toml

```diff
@@ -200,6 +200,10 @@ path = "examples/3d/update_gltf_scene.rs"
 name = "wireframe"
 path = "examples/3d/wireframe.rs"
 
+[[example]]
+name = "chess"
+path = "examples/3d/chess.rs"
```

Comment:
I think it should probably be renamed to "chess_render" or something, to emphasize from the title that it's not interactive at all.

Reply:
Pending other comments, I iterate on this comment shortly.